RPLCD/i2c.py

```diff
@@ -32,15 +32,40 @@ class CharLCD(BaseCharLCD):
     def __init__(self, address, port=1,
                        cols=20, rows=4, dotsize=8,
                        auto_linebreaks=True,
-                       backlight_enabled=True):
+                       backlight_enabled=True, i2c_expander="PCF8574"):
```

Comment:
Hm, you're right. I should have merged this before the PyPI release, then we wouldn't have any reverse compatibility issues.

On the other hand the PCF8574 is quite widely used, so it's probably a good default.